src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -93,17 +94,24 @@ private static boolean isValidPath(Path filePath) {
      */
     public void save(AddressBook addressBook) throws StorageOperationException {
 
+        File storageFile = path.toFile();
+        if(!storageFile.exists()){
+            throw new StorageOperationException("Storage file not found!");           
```

Comment:
can create another exception extends from StorageOperationException, when creating an exception, we want the exception name as clear as possible for example file not found exception is clearly about file not found. :)